react/features/toolbox/components/Toolbox.native.js

```diff
@@ -119,15 +120,25 @@ class Toolbox extends Component {
      * @returns {ReactElement}
      */
     render() {
+        if (!this.props._visible) {
```

Comment:
why is this necessary? this is already implemented at the container level, isn't it?

Reply:
it doesn't work as expected - see Container implementation - it barley adjusts width and height which may not always be respected by child components

Reply:
then we should fix it there IMHO. Note that now that the filmstrip is always visible we can remove the workaround and return null on all platforms.

Reply:
ok

Reply:
but on the other hand, why even bother with doing the whole render logic if there's nothing to be rendered ?